[2024-01-15T14:45:55.045-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T17:44:40.621620+00:00 [queued]>
[2024-01-15T14:45:55.048-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T17:44:40.621620+00:00 [queued]>
[2024-01-15T14:45:55.048-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-15T14:45:55.061-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-15 17:44:40.621620+00:00
[2024-01-15T14:45:55.065-0300] {standard_task_runner.py:57} INFO - Started process 62940 to run task
[2024-01-15T14:45:55.067-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_novo', 'df', 'scheduled__2024-01-15T17:44:40.621620+00:00', '--job-id', '1027', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpxpha9d2d']
[2024-01-15T14:45:55.067-0300] {standard_task_runner.py:85} INFO - Job 1027: Subtask df
[2024-01-15T14:45:55.098-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_novo.df scheduled__2024-01-15T17:44:40.621620+00:00 [running]> on host desktop
[2024-01-15T14:45:55.153-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_novo' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T17:44:40.621620+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T17:44:40.621620+00:00'
[2024-01-15T14:45:55.646-0300] {logging_mixin.py:154} INFO - Called | 1705340755.6438556 | df
[2024-01-15T14:45:55.651-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-15T14:45:55.652-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-15T14:45:55.654-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_novo, task_id=df, execution_date=20240115T174440, start_date=20240115T174555, end_date=20240115T174555
[2024-01-15T14:45:55.679-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-15T14:45:55.686-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
